calendar/calendar-impl/impl/src/java/org/sakaiproject/calendar/impl/BaseCalendarService.java

```diff
@@ -5737,6 +5729,16 @@ protected void handleAccessIcalCommon(HttpServletRequest req, HttpServletRespons
 			log.warn("Reject internal request for: "+ calRef);
 			return;
 		}
+
+		// Extract the alias name to use for the filename.
+		List alias =  m_aliasService.getAliases(calRef);
```

Comment:
add generics i.e. `List<Alias>`

Reply:
I didn't "really" make this change, just moved the existing code around, but I can still do it.

Reply:
Change made.